### DISCUSSION TITLE
Is there a plan to back port the V8 compilation cache fix to v18.x?

### DISCUSSION THREAD

**Discussion by opjarretop** [nodejs/help#4693] [Q&A]
I recently came across this caching issue and found that it has been fixed in v21.1.0 and now is being back ported to v20 https://github.com/nodejs/node/pull/50682. Is there a plan for a v18 patch to include this fix as well?

**Comment by atlowChemi** [comment]
Unfortunately most likely it will not reach node v18, as v18 is now in maintenance mode.
You can see [this backport PR](https://github.com/nodejs/node/pull/51004) which has been opened for v18.
